dist/monapt.d.ts

```diff
@@ -1,182 +1,180 @@
-        toOption(): Option<T>;
-    }
-    var Try: <T>(f: () => T) => Try<T>;
-    interface ICrackerProducer<F> {
-        (f: F): void;
-    }
-    class Cracker<F extends Function> {
-        private fired;
-        private callbacks;
-        private producer;
-        fire(producer: ICrackerProducer<F>): void;
-        private fireAll();
-        add(fn: F): void;
-    }
-    interface ICompleteFunction<T> {
-        (trier: Try<T>): void;
-    }
-    interface IFutureSuccess<T> {
-        (value: T): void;
-    }
-    interface IFutureFailure<T> {
-        (error: Error): void;
-    }
-    interface IFuturePromiseLike<T> {
-        (value: T): void;
-        (error: Error): void;
-        success: IFutureSuccess<T>;
-        failure: IFutureFailure<T>;
-    }
-    class Future<T> {
-        private cracker;
-        constructor(future: (promise: IFuturePromiseLike<T>) => void);
-        static succeed<T>(value: T): Future<T>;
-        static failed<T>(error: Error): Future<T>;
-        success(value: T): void;
-        failure(error: Error): void;
-        onComplete(callback: ICompleteFunction<T>): void;
-        onSuccess(callback: (value: T) => void): void;
-        onFailure(callback: (error: Error) => void): void;
-        map<U>(f: (value: T, promise: IFuturePromiseLike<U>) => void): Future<U>;
-        flatMap<U>(f: (value: T) => Future<U>): Future<U>;
-        filter(predicate: (value: T) => boolean): Future<T>;
-        reject(predicate: (value: T) => boolean): Future<T>;
-        recover(fn: (e: Error, promise: IFuturePromiseLike<T>) => void): Future<T>;
-        recoverWith(fn: (e: Error) => Future<T>): Future<T>;
-    }
-    class Promise<T> extends Future<T> {
-        isComplete: boolean;
-        constructor();
-        success(value: T): void;
-        failure(error: Error): void;
-        future(): Future<T>;
-    }
-    var future: <T>(f: (promise: IFuturePromiseLike<T>) => void) => Future<T>;
-    interface IHashable {
-        hash?(): string;
-    }
-    class Map<K extends IHashable, V> {
-        private real;
-        private selector;
-        constructor(key: K, value: V, ...keysAndValues: Array<any>);
-        constructor(raw: Object);
-        constructor();
-        private ensureSelector(hint?);
-        private add(key, value);
-        foreach(f: (key: K, value: V) => void): void;
-        map<K2, V2>(f: (key: K, value: V) => Tuple2<K2, V2>): Map<K2, V2>;
-        flatMap<K2, V2>(f: (key: K, value: V) => Map<K2, V2>): Map<K2, V2>;
-        mapValues<U>(f: (value: V) => U): Map<K, U>;
-        filter(predicate: (key: K, value: V) => boolean): Map<K, V>;
-        reject(predicate: (key: K, value: V) => boolean): Map<K, V>;
-        find(f: (key: K, value: V) => boolean): Option<Tuple2<K, V>>;
-        get(key: K): Option<V>;
-        getOrElse(key: K, defaultValue: () => V): V;
-        head(): Option<Tuple2<K, V>>;
-    }
+export interface Tuple1<A> {
```

Comment:
Ah, so it's still like that :(
Can you remove the change to the typings file?  ^^  It's too easy for it to get out-of-sync with `master`, and I usually do so before releasing.